homeassistant/components/proxmox/binary_sensor.py

```diff
@@ -0,0 +1,54 @@
+import homeassistant.components.proxmox as proxmox
+from homeassistant.components.switch import SwitchDevice
+
+DOMAIN = 'proxmox'
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up Proxmox VE binary sensor."""
+    nodes = hass.data[proxmox.DATA_PROXMOX_NODES]
+    sensors = []
+
+    for node_name in nodes.keys():
+        item = nodes[node_name]
+        if 'type' not in item or item['type'] != 'node':
+            sensors.append(PXMXBinarySensor(hass, node_name, 'Is Running', lambda node: 'running' == node['status']))
+
+    async_add_entities(sensors)
+
+
+class PXMXBinarySensor(SwitchDevice):
+
+    def __init__(self, hass, node_name, sensor_name, sensor_value):
+        """Initialize Proxmox VE binary sensor."""
+        self._hass = hass
+        self._node_name = node_name
+        self._sensor_name = sensor_name
+        self._sensor_value = sensor_value
+        self.update()
```

Comment:
This is [obsolete](https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#5-entity).

Reply:
Fixed.